packages/rocketchat-message-snippet/client/lib/collections.js

```diff
@@ -0,0 +1 @@
+this.Messages = new Meteor.Collection('rocketchat_message');
```

Comment:
And, can you change the variable name? This variable will store only messages from snippeted messages and only when you have the tab bar open